docs/v1/guidance-for-generating-p4info.md

```diff
@@ -0,0 +1,363 @@
+# Guidance for generating P4Info messages
+
+The P4Runtime v1.0 specification restricts the types that it supports
+for the following kinds of things:
+
++ table search key fields, defined in the P4Info message in a
+  `MatchField` message
++ fields of a `ValueSet`, also defined in the P4Info message in a
+  `MatchField` message
++ parameters specified by the control plane for a table action,
+  defined in the P4Info message in a `Param` message
++ metadata fields in a header sent from data plane to controller, or
+  from controller to the data plane, defined in the P4Info message in
+  a `Metadata` message, if a recently proposed [p4runtime PR
```

Comment:
you may want to update this now that the PR has been merged, although it may be good to mention that it is not available in P4Runtime v1.0.0 (but will be in v1.0.1).

Reply:
I will change the text to this: "defined in the P4Info message in
  a `Metadata` message (added in P4Runtime version 1.0.1)"